robotbase/src/main/java/com/team2357/frc2022/Constants.java

```diff
@@ -147,6 +147,11 @@
         public static final String ARDUINO_SENSOR_DEVICE_NAME = "/dev/ttyACM0";
     }
 
+    public final class SENSORS {
```

Comment:
Let's rename this ```DIO_IDS``` and put the drive encoder IDs in here as well. This way all of the IDs for devices that connect via the RobRio's DIO ports are in one place.